### ISSUE TITLE
Missing empty alt text for SVG icons

### ISSUE THREAD

**Issue by wenche** [equinor/energyvision#970]
![Skjermbilde 2022-04-05 kl. 10.20.20.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/ee471b81-45e2-4c4e-9255-dbf105943c9a)

### Acceptance criteria
1. Should have an empty alt tag
1. Should have `role=presentation` to reduce noice in tooling

**Comment by wenche** [comment]
We had missed this one @SvSven  should be OK now :fingers_crossed:

**Comment by SvSven** [comment]
Hmm, I thought the empty alt tag was enough and the `role="presentation"` wasn't needed.
Can confirm that on Edge we have `alt` and `role="presentation"` and on FireFox we have `alt=""` and `role="presentation"` 👍
